tsconfig.json

```diff
@@ -1,9 +1,18 @@
 {
   "compilerOptions": {
+    "rootDir": "./src",
     "strict": true,
-    "module": "commonjs",
-    "target": "es2016",
+    "module": "esnext",
+    "moduleResolution": "node",
+    "esModuleInterop": true,
```

Comment:
thoughts on keeping this `false` (or removing it)? It has potential downstream effects for consumers

Reply:
Not sure, could you expand on that or post a link that describes the effects you're thinking of? I've needed it in application code before but I'm not sure what the implications are for turning it on in a library